app/decorators/order_detail_decorator.rb

```diff
@@ -0,0 +1,6 @@
+class OrderDetailDecorator < SimpleDelegator
+
+  delegate :full_name, to: :assigned_user, prefix: :assigned_user, allow_nil: true
+  delegate :name, to: :order_status, prefix: :order_status, allow_nil: true
```

Comment:
I think it might be worth having a conversation with Dan since he has a similar--but more full featured-- presenter in #397.